asset/quickstart/create-feed/main_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	asset "cloud.google.com/go/asset/apiv1p2beta1"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+	assetpb "google.golang.org/genproto/googleapis/cloud/asset/v1p2beta1"
+)
+
+func TestMain(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	os.Setenv("GOOGLE_CLOUD_PROJECT", tc.ProjectID)
+	ctx := context.Background()
+	cloudresourcemanagerClient, err := cloudresourcemanager.NewService(ctx)
+	if err != nil {
+		t.Fatalf("cloudresourcemanager.NewService: %v", err)
+	}
+
+	project, err := cloudresourcemanagerClient.Projects.Get(tc.ProjectID).Do()
+	if err != nil {
+		t.Fatalf("cloudresourcemanager.Projects.Get.Do: %v", err)
+	}
+	projectNumber := strconv.FormatInt(project.ProjectNumber, 10)
+
+	m := testutil.BuildMain(t)
+	defer m.Cleanup()
+
+	if !m.Built() {
+		t.Errorf("failed to build app")
+	}
+
+	stdOut, stdErr, err := m.Run(nil, 30*time.Second, fmt.Sprintf("--project_id=%s", tc.ProjectID))
```

Comment:
I don't think the `--project_id` flag is used?

Pass `GOOGLE_CLOUD_PROJECT` as the first arg rather than using `os.Setenv`?

```
env := map[string]string{"GOOGLE_CLOUD_PROJECT": tc.ProjectID}
m.Run(env, ...)
```

Reply:
Done, thanks for the suggestions.